### PR TITLE
Test model product,supplier,import,order

### PR DESCRIPTION
**Pull request by hatcaiday**
https://edu-redmine.framgia.vn/issues/5978
screenshot from 2019-01-23 16-36-07

### CODE REVIEW

**Comment by hatcaiday** [comment]
ready

File: app/models/import_detail.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class ImportDetail < ApplicationRecord
   has_many :products
   belongs_to :import
+  validates :price, presence: true
```
Comment (huongnguyenmta) [inline app/models/import_detail.rb]: thêm validate number, min, max cho price nữa nhé em

File: app/models/order.rb [hunk 2]
```diff
@@ -1,4 +1,9 @@
 class Order < ApplicationRecord
   has_many :order_details
   belongs_to :user
+  validates :receiver, presence: true
+  validates :order_place, presence: true
+  validates :order_phone, presence: true
+  validates :status, presence: true
```
Comment (huongnguyenmta) [inline app/models/order.rb:7]: những status nào thế e?

**Comment by hatcaiday** [comment]
ready
test

**Pull request merged**